generate_export_options.rb

```diff
@@ -98,6 +98,7 @@ def export_method(mobileprovision_content)
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+export_options[:uploadBitcode] = false
 
```

Comment:
Ok, thanks a bunch. I'll leave it for now.
I briefly discussed an idea with Krisztian:
Based on an toggle option:
Either zip the app.dSYM as today 
Or zip all dSYMs into one big zip-file.

But in both cases export the path to the zip as BITRISE_DSYM_PATH

This ought to make the script directly compatible with uploading multiple dSYMs to HockeyApp in one go (based on a discussion link that Krisztian sent me: https://support.hockeyapp.net/discussions/problems/29779-symbolication-on-ios8-framework )
I haven't performed this experiment myself yet (since we have a big release coming up), but hopefully I will get to it next week.
If this works, would it be a toggle that you could be willing to add to the official Xcode Archive step?